manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -216,6 +216,8 @@ data:
       "experimentsURL" : "http://kruize.monitoring.svc.cluster.local:8080/createExperiment",
       "experimentNameFormat" : "%datasource%|%clustername%|%namespace%|%workloadname%(%workloadtype%)|%containername%",
       "bulkapilimit" : 1000,
+      "isKafkaEnabled" : "true",
+      "kafkaBootstrapServers" : "kruize-kafka-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092",
```

Comment:
is it possible to set via env variable ?

Reply:
Created #1497 to track this

Reply:
@khansaad Are you addressing setting the kafka bootstrap server using env in this PR?

Reply:
> is it possible to set via env variable ?

Replaced the server details config value with env variable now